### PR TITLE
topsql: add aggregate by table or db

### PR DESCRIPTION
**Pull request by nolouch**
### What is changed, added or deleted? (Required)

Corresponding PR for docs in English: https://github.com/pingcap/docs/pull/18982

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):** 

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): https://github.com/pingcap/tidb/issues/55540

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/top-sql.md [hunk 1]
```diff
@@ -71,31 +67,39 @@ SET GLOBAL tidb_enable_top_sql = 1;
...
 
-除此以外：
+    你可以在时间选择器中调整时间范围，或在图表中框选一个时间范围，来更精确、细致地观察问题。更小的时间范围将能提供细节更丰富的数据，数据最高精度可达 1 秒。
+
+    ![修改时间范围](/media/dashboard/top-sql-usage-change-timerange.png)
+
+    如果图表中显示的数据已过时，你可以点击**刷新** (Refresh) 按钮，或在**刷新** (Refresh) 下拉列表中选择自动刷新。
+
+    ![刷新](/media/dashboard/top-sql-usage-refresh.png)
+
```
Comment (lilin90) [inline dashboard/top-sql.md:93]: 注：1-5 步的内容只调整了下格式。

**Review by crazycs520 — APPROVED** [review]

File: dashboard/top-sql.md [hunk 2]
```diff
@@ -25,11 +25,7 @@ Top SQL 适用于分析性能问题。以下列举了一些典型的 Top SQL 适
 
 Top SQL 不能用于解答与性能无关的问题，例如数据正确性或异常崩溃问题。
```
Comment (hfxsd) [inline dashboard/top-sql.md:26]:
```suggestion
```

File: dashboard/top-sql.md [hunk 3]
```diff
@@ -25,11 +25,7 @@ Top SQL 适用于分析性能问题。以下列举了一些典型的 Top SQL 适
...
 
-当前 Top SQL 仍然处于早期阶段，功能正在持续加强。以下列举了一些目前暂不支持的场景，供参考：
-
-* 暂时不支持分析 Top 5 以外 SQL 语句的开销情况（如多业务混合时）。
-* 暂时不支持按 User、Database 等不同维度分析 Top N SQL 语句的开销情况。
-* 暂时不支持分析并非由于 CPU 负载高导致的数据库性能问题，例如锁冲突。
+Top SQL 暂不支持分析并非由于 CPU 负载高导致的数据库性能问题，例如锁冲突。
```
Comment (hfxsd) [inline dashboard/top-sql.md:28]:
```suggestion
Top SQL 不适用于分析以下问题：

- 不能用于解答与性能无关的问题，例如数据正确性或异常崩溃问题。
- Top SQL 暂不支持分析并非由于 CPU 负载高导致的数据库性能问题，例如锁冲突。
```

**Review by hfxsd — APPROVED** [review]

File: dashboard/top-sql.md [hunk 4]
```diff
@@ -71,31 +68,39 @@ SET GLOBAL tidb_enable_top_sql = 1;
...
+    ![修改时间范围](/media/dashboard/top-sql-usage-change-timerange.png)
+
+    如果图表中显示的数据已过时，你可以点击**刷新** (Refresh) 按钮，或在**刷新** (Refresh) 下拉列表中选择自动刷新。
+
+    ![刷新](/media/dashboard/top-sql-usage-refresh.png)
+
+6. 查看 TiKV 按表或者数据库维度的 CPU 资源使用情况，快速定位更高维度的资源使用情况。
 
-* 你可以在时间选择器中调整时间范围，或在图表中框选一个时间范围，来更精确、细致地观察问题。更小的时间范围将能提供细节更丰富的数据，数据最高精度可达 1 秒。
+    选择一个 TiKV 实例，然后选择 **By TABLE** 或者 **By DB**：
```
Comment (nolouch) [inline dashboard/top-sql.md:97]:
```suggestion
    选择一个 TiKV 实例，然后选择 **By TABLE** 或者 **By DB** （目前只支持 TiKV 实例选择聚合维度）：
```

File: dashboard/top-sql.md [hunk 5]
```diff
@@ -71,31 +68,39 @@ SET GLOBAL tidb_enable_top_sql = 1;
...
-除此以外：
+    你可以在时间选择器中调整时间范围，或在图表中框选一个时间范围，来更精确、细致地观察问题。更小的时间范围将能提供细节更丰富的数据，数据最高精度可达 1 秒。
+
+    ![修改时间范围](/media/dashboard/top-sql-usage-change-timerange.png)
+
+    如果图表中显示的数据已过时，你可以点击**刷新** (Refresh) 按钮，或在**刷新** (Refresh) 下拉列表中选择自动刷新。
+
+    ![刷新](/media/dashboard/top-sql-usage-refresh.png)
+
+6. 查看 TiKV 按表或者数据库维度的 CPU 资源使用情况，快速定位更高维度的资源使用情况。
```
Comment (lilin90) [inline dashboard/top-sql.md:95]:
```suggestion
6. 查看按表或者数据库维度的 CPU 资源使用情况，快速定位更高维度的资源使用情况。目前只支持查看 TiKV 实例。
```

**Review by songrijie — APPROVED** [review]

**Review by lilin90 — COMMENTED** [review]
/approve

**Pull request merged**